test/configs/zombieTanssiWarpSync.json

```diff
@@ -0,0 +1,104 @@
+{
+    "settings": {
+      "timeout": 1000,
+      "provider": "native"
+    },
+    "relaychain": {
+      "chain": "rococo-local",
+      "default_command": "tmp/polkadot",
+      "default_args": [
+        "--no-hardware-benchmarks",
+        "-lparachain=debug",
+        "--database=paritydb"
+      ],
+      "nodes": [
+        {
+          "name": "alice",
+          "ws_port": "9947",
+          "validator": true
+        },
+        {
+          "name": "bob",
+          "validator": true
+        },
+        {
+          "name": "charlie",
+          "validator": true
+        },
+        {
+          "name": "dave",
+          "validator": true
+        }
+      ]
+    },
+    "parachains": [
+      {
+        "id": 1000,
+        "chain_spec_path": "specs/tanssi-1000.json",
+        "collators":  [
+          {
+            "name": "Collator1000-01",
+            "ws_port": "9948",
+            "command": "../target/release/tanssi-node"
+          },
+          {
+            "name": "Collator1000-02",
+            "command": "../target/release/tanssi-node"
+          },
+          {
+            "name": "Collator2000-01",
+            "command": "../target/release/tanssi-node"
+          },
+          {
+            "name": "Collator2000-02",
+            "command": "../target/release/tanssi-node"
+          },
+          {
+            "name": "Collator2001-01",
+            "command": "../target/release/tanssi-node"
+          },
+          {
+            "name": "Collator2001-02",
+            "command": "../target/release/tanssi-node"
+          },
+          {
+            "name": "Collator1000-03",
+            "command": "../target/release/tanssi-node"
+          }
+        ]
+      },
+      {
+        "id": 2000,
+        "chain_spec_path": "specs/template-container-2000.json",
+        "collators":  [
+          {
+            "name": "FullNode-2000",
+            "validator": false,
+            "command": "../target/release/container-chain-template-simple-node",
+            "ws_port": 9949,
+            "p2p_port": 33049
+          }
+        ]
+      },
+      {
+        "id": 2001,
```

Comment:
We dont need to containrs to check this right? what about just using one and potentially running these in ubuntu-latest?

Orchestrator: A,B ,E
Container2000: C,D

Delete D, E should go to container2000 with warp sync

Reply:
Correct, container 2001 is unused, I can remove it.